dipy/reconst/dti.py

```diff
@@ -768,10 +768,6 @@ def fit(self, data, mask=None, adjacency=False):
         mask : array
             A boolean array used to mark the coordinates in the data that
             should be analyzed that has the shape data.shape[:-1]
-
-        adjacency : float, optional
-            Calculate voxel adjacency accounting for mask, using this
-            value as cutoff distance (measured in voxel coordinates)
```

Comment:
It should normally have a deprecation cycle since it is already in the previous release and we do not know if someone is using it already

Reply:
I really doubt anyone is using it, because it is not being used in any fitting functions (it's very unlikely someone is currently using it, but if they are _developing_ something then they will be able to adapt, I think). I realized that having the adjacency calculation happen here, was probably not a good idea (I added it here in the first place).

Reply:
ok, fine 👍

Reply:
Thanks, sorry for the change